clients/python/moondream/server.py

```diff
@@ -0,0 +1,202 @@
+                            )["answer"]:
+                                self.stream_tokens(tokens, completed=False)
+                            self.stream_tokens("", completed=True)
+                        except Exception as e:
+                            logger.error("Error during query streaming", exc_info=True)
+                            self.stream_tokens(
+                                "An error occurred during query processing.",
+                                completed=True,
+                            )
+                    else:
+                        result = self.model.query(image, question)
+                        self.send_json_response(result)
+                except Exception as e:
+                    logger.error("Query processing error", exc_info=True)
+                    self.send_error_response("Query processing failed.")
+
+            elif endpoint == "/detect":
+                try:
+                    object_name = payload.get("object")
+                    if not object_name:
+                        raise ValueError("object is required")
+                    result = self.model.detect(image, object_name)
+                    self.send_json_response(result)
+                except Exception as e:
+                    logger.error("Object detection error", exc_info=True)
+                    self.send_error_response("Object detection failed.")
+
+            elif endpoint == "/point":
+                try:
+                    object_name = payload.get("object")
+                    if not object_name:
+                        raise ValueError("object is required")
+                    result = self.model.point(image, object_name)
+                    self.send_json_response(result)
+                except Exception as e:
+                    logger.error("Object pointing error", exc_info=True)
+                    self.send_error_response("Object pointing failed.")
+
+        except Exception as e:
+            logger.error("Unexpected error in request handling", exc_info=True)
+            self.send_error_response("An unexpected error occurred.")
+
+    def do_OPTIONS(self) -> None:
+        self.send_response(200)
+        self.send_header("Access-Control-Allow-Origin", "*")
+        self.send_header("Access-Control-Allow-Methods", "POST, OPTIONS")
+        self.send_header("Access-Control-Allow-Headers", "Content-Type")
+        self.end_headers()
+
+    def do_GET(self) -> None:
+        if self.path == "/health" or self.path == "/":
+            self.send_response(200)
+            self.send_header("Content-Type", "application/json")
+            self.send_header("Access-Control-Allow-Origin", "*")
+            self.end_headers()
+            self.wfile.write(
+                json.dumps(
+                    {"status": "ok", "message": "Moondream server is running!"}
+                ).encode()
+            )
+        else:
+            self.send_error_response("Method not allowed", 405)
```

Comment:
Let's remove `/health`... it's not needed... and instead return some useful HTML content with a link to the docs if the user opens the URL in their web browser.